library/core/registry/src/main/java/org/quiltmc/qsl/registry/mixin/client/ItemRendererAccessor.java

```diff
@@ -0,0 +1,21 @@
+package org.quiltmc.qsl.registry.mixin.client;// Created 2022-04-12T11:27:59
```

Comment:
```suggestion
package org.quiltmc.qsl.registry.mixin.client;
```

And please run `gradle :core:registry:applyLicenses`

Reply:
Oh, forgot about the timestamp. Seems I need to remove that for QSL in case I ever need to PR new classes lol